src/streams/StreamIcon.js

```diff
@@ -14,17 +14,24 @@ type Props = $ReadOnly<{|
   style?: TextStyleProp,
 |}>;
 
-export default ({ color, style, isPrivate, isMuted, isWebPublic, size }: Props): Node => {
-  let StreamIcon = undefined;
+export default function StreamIcon({
+  color,
+  style,
+  isPrivate,
+  isMuted,
+  isWebPublic,
+  size,
+}: Props): Node {
```

Comment:
nit: perhaps a good case for the `const { color, style, isPrivate, isMuted, isWebPublic, size } = props;` form instead, particularly if that gets it back to one line